package/upgrade/upgrade_manifests.sh

```diff
@@ -485,17 +552,16 @@ upgrade_logging_event_audit() {
     # rancher-logging_event-extension is reusing chart rancher-logging, but as an extension for event
     upgrade_managed_chart_from_version $UPGRADE_PREVIOUS_VERSION rancher-logging_event-extension rancher-logging.yaml
 
-    echo "Apply resource file"
+    echo "Apply resource file of event"
 
     kubectl apply -f ./rancher-logging.yaml
 
     # wait few seconds
     sleep 20
```

Comment:
ditto, do we need to sleep 20 seconds here?

Reply:
similar consideration